resources/lib/ytchannels.py

```diff
@@ -279,6 +282,10 @@ def ytchannels_main():
 			url = build_url({'mode': 'open_channel', 'foldername': '%s'%id, 'page':'1', 'playlist':'yes'})
 			li = xbmcgui.ListItem('%s'%name)
 			li.setArt({'icon':'%s'%thumb})
+			add_playlist_uri = build_url({'mode': 'add_playlist', 'id': '%s'%id, 'name': '%s'%name, 'thumb': '%s'%thumb})
+			items = []
+			items.append(("Add to folder", 'RunPlugin(%s)'%add_playlist_uri))
```

Comment:
I can't figure out how to get to this? Is the intention that I should be able to add a given playlist into its own folder? I can see playlists have been added in yellow etc, however I can't find this menu anywhere. I only have "Add to favourites" if I rightclick a given playlist

Reply:
If you have playlists enabled, when you browse to a channel it has Playlists in yellow at the top of each page (this functionality was already there). If you look at the playlists, you can open the context menu and choose Add to folder. Then the playlist will show up along with the channels in that folder. It's useful if a channel you watch a lot of specific playlists for different topics, instead of having to navigate to the playlists of that channel it's right there in the folder.

Reply:
> OK, stupid question time: how do I actually use the new functionality?
> 
> Edit: I found it; I'd messed up deploying the updated code to my test installation
> Ediit2: Still don't quite understand what functionality it is adding other than seeing playlists?

Seeing playlists has always been there, this just allows you to add a playlist directly to a folder, just like you can add a channel to a folder. It just cuts down on having to navigate to a channel then find the playlist in the list of playlists if there is a playlist you check often.